### ISSUE TITLE
Syntax errors in example.py

### ISSUE THREAD

**Issue by israel-fl** [AHAAAAAAA/PokemonGo-Map#693]
GitHub issues are for bug reporting and feature requests.  
For general support, join [our discord server](https://discord.gg/Kzp4yky).
To make bug reports easy to find and organized, we close issues that are deemed
out of scope for GitHub Issues and point people to Discord.

Please provide the following information.
The more information you provide, the more easily we will be able to offer
help and advice.

### Environment info
Operating System:
Ubuntu 14.04
Installed version of Python:
Python 3.4

### Steps to reproduce
1. Simply run example.py
2.
3.

### What have you tried?
1. Fixing the syntax!

I'm not sure if it's because I'm running the interpreter in Linux, or some other issue, but the interpreter is unable to run the python file. My python proficiency is limited, but most of the syntax errors had to do with the print statements.

print 'Some_output'.format...

Should instead be 

print ('Some_output'.format...)

Basically there's a lot of print statements missing parenthesis. There's also functions without proper indentation (I believe it was around lines 200)

Another thing, on line 840 (I think) there's html <div><b>Name</b> etc...
Traceback prints a syntax error, I'm not sure if the html is there on purpose.

### Logs or other output that would be helpful
(If logs are large, please upload as attachment).

**Comment by medghaim** [comment]
Sorry, but as per the first two lines in the README.md we've dropped support for the `master` branch. Please switch branches to `develop` :)

**Comment by medghaim** [comment]
Also, this won't work with Python 3.4. Try 2.7
